elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -1086,6 +1086,30 @@ See mixin blocks before selectors that contain the selector name and the three l
     @include top-level-button--stays-in-top-bar;
   }
 }
+// @todo: (KS) add styles to hide account toggle and show the account link when logged-in class does NOT exist
+.pfe-navigation__account-wrapper {
+  
+  // Hide account toggle if NOT logged in
+  // .pfe-navigation__account-toggle {
+  //   display: none;
+  // }
+  // // Show login link if NOT logged in
+  // .pfe-navigation__log-in-link {
+  //   display: flex;
+  // }
+
+  .pfe-navigation__account-wrapper--logged-in {
```

Comment:
```suggestion
  &--logged-in {
```
^ This is a nice option in Sass and reduces the specificity generated by stacking selectors.

Reply:
The `[class][class][class]` override for example might not be needed to override specificity if styles are attached to a single class rather than a class stack.

Reply:
I'd prefer not to do string concatenation in selectors, it makes things harder to find in the codebase, but I agree on the specificty if it isn't necessary.

Reply:
Can you please share more about what's wrong with string concatenation?  It's how ever other component in the system is written and is extremely common practice in Sass.  It's also, I would suggest, when done correctly, a much cleaner and easier way to read and find selectors.  Especially if you don't nest them more than 1 or 2 levels deep.

Reply:
I will add, it lets you roll up code based on regions in a more logical way so you can easily get to the sections you are interested in for debugging purposes.

<img width="1023" alt="Screen Shot 2021-03-19 at 11 38 53 AM" src="https://user-images.githubusercontent.com/1840295/111805786-bb67f380-88a7-11eb-8a10-807b0b699a58.png">

Reply:
My issue is if I want to find `.pfe-card__footer`, searching for it with a find won't work. I also prefer to try and keep Sass declaration blocks on the small side so you can tell what styles roll up to, and this encourages larger declaration blocks.